### ISSUE TITLE
Introduce Ember CLI AddonDocs

### ISSUE THREAD

**Issue by raido** [adopted-ember-addons/ember-page-title#194]
Migrate to https://ember-learn.github.io/ember-cli-addon-docs/docs from current basic documentation page.

This provides consistency with other addons and flexibility to easily add new information.

**Comment by raido** [comment]
There are talks of deprecating ember-cli-addon-docs.

**Comment by knownasilya** [comment]
Yeah, have seen that coming for a while. An alternative might be https://docfy.dev/ but it's a little young and doesn't support versioned docs. Really we should be able to use any docs generator in the JS ecosystem, esp one that supports TS types since that is the proposed official doc strategy https://github.com/emberjs/rfcs/pull/678

cc @gossi @josemarluedke
